scipy/special/_hyp0f1.pxd

```diff
@@ -115,8 +116,10 @@ cdef inline double complex _hyp0f1_cmplx(double v, double complex z) nogil:
         return 1.0
 
     # both v and z small: truncate the Taylor series at O(z**2)
-    if zabs(z) < 1e-6*(1.0 + fabs(v)):
-        return 1.0 + z/v + z*z/(2.0*v*(v+1.0))
+    if zabs(z) < 1e-6*(1.0 + zabs(v)):
```

Comment:
I would have said - maybe worth adding comment here, so no-one reverts this later.